global-privacy-control/config_reference.json

```diff
@@ -0,0 +1,21 @@
+{
+    "readme": "This configuration has gpc feature only with some exceptions",
+    "features": {
+        "gpc": {
+            "state": "enabled",
+            "exceptions": [
+                {
+                    "domain": "washingtonpost.com",
+                    "reason": "This is one of the domains where our iOS/Android apps do inject hte"
```

Comment:
```suggestion
                    "reason": "This is a domain where our iOS/Android apps inject the Sec-GPC request header"
```

Reply:
oh, ops, I must have edited it by a mistake, thanks for catching!